dynamo/configuration.py

```diff
@@ -62,16 +63,18 @@ def gen_layer_pearson_residual_key(layer: str):
         )
 
     def select_layer_data(adata: AnnData, layer: str, copy=False) -> pd.DataFrame:
-        """select layer data based on layer key. The default layer is X layer in adata.
-        For layer-like data such as X stored in adata.X (but not in adata.layers) and protein data specified by dynamo convention,
-        this utility provides an unified interface for selecting layer data with shape n_obs x n_var."""
+        """select layer data based on layer key. The default layer is X layer in
+        adata. For layer-like data such as X stored in adata.X (but not in
+        adata.layers) and protein data specified by dynamo convention, this
+        utility provides an unified interface for selecting layer data with
+        shape n_obs x n_var."""
```

Comment:
this utility provides an unified interface for selecting layer data.  
        The default layer is X layer in
        adata with shape n_obs x n_var. For protein data it selects adata.obsm["protein"] as specified by dynamo convention (the number of proteins are generally less than detected genes `n_var`).  For other layer data, select data based on layer key with shape n_obs x n_var